### PR TITLE
Output merge improvements

### PR DESCRIPTION
**Pull request by martinal**
Tries to mark outputs better based on diffs instead of throwing all outputs between markers with output-strategy inline.

Also please review the changes in diff for attachments, these are untested.

### CODE REVIEW

**Comment by vidartf** [comment]
There should be some attachment notebooks in the test files folder, (hand crafted).

**Comment by martinal** [comment]
Don't merge this, I'm working on fixes in the output compare predicates.

**Comment by martinal** [comment]
Ready for review now. Includes an attempt at conflict handling for attachments for inline mode.

There's a bug in decision bundling which affects attachments, I'll submit an issue on that separately since attachments are experimental anyway.

File: nbdime/merging/autoresolve.py [hunk 1]
```diff
@@ -562,6 +625,121 @@ def make_inline_source_decision(source, prefix, local_diff, remote_diff):
     )]
 
 
+def make_inline_attachments_decision(attachments, prefix, local_diff, remote_diff):
```
Comment (martinal) [inline nbdime/merging/autoresolve.py]: @vidartf here (mentioned in https://github.com/jupyter/nbdime/issues/197)

**Pull request merged**